client/my-sites/stats/stats-banners/index.jsx

```diff
@@ -112,15 +114,24 @@ class StatsBanners extends Component {
 	}
 
 	render() {
-		const { gsuiteDomainName, isCustomerHomeEnabled, planSlug, siteId, domains } = this.props;
+		const {
+			gsuiteDomainName,
+			isCustomerHomeEnabled,
+			isAllowedToManageSite,
+			planSlug,
+			siteId,
+			domains,
+		} = this.props;
 
 		if ( isEmpty( domains ) ) {
 			return null;
 		}
 
 		return (
 			<Fragment>
-				{ gsuiteDomainName && <QueryEmailForwards domainName={ gsuiteDomainName } /> }
+				{ gsuiteDomainName && isAllowedToManageSite && (
+					<QueryEmailForwards domainName={ gsuiteDomainName } />
+				) }
```

Comment:
I could see that block moved to the [`renderGSuiteBanner()` method](https://github.com/Automattic/wp-calypso/blob/088eeb9a748199941439fa503e4db2fa551aafbc/client/my-sites/stats/stats-banners/index.jsx#L80). This would make clearer that we only need to fetch email forwards when we display the G Suite banner. I'm worried that this data may actually be required by another banner such as `WpcomChecklist` though. What do you think?

Reply:
I see what you mean. However `GSuiteStatsNudge` is conditionally displayed based on email forwards data which needs to be fetched in advance.